drivers/hmis_csv_importer/app/views/hmis_csv_importer/import_overrides/_table.haml

```diff
@@ -0,0 +1,20 @@
+%table.table.table-striped
+  %thead
+    %tr
+      %th File
+      %th Replaces
+      %th With
+      %th When
+      - if editable
+        %th
+  %tbody
+    - overrides.each do |override|
+      %tr
+        %td= override.file_name
+        %td= override.replaces_column
+        %td= override.describe_with
```

Comment:
I don't know if your screenshot is just out of date, but it seems overly wordy - `replaces ProjectType with 8` when that is already described by the column headers.

Maybe cleaner would be
```
File, Attribute, Value, Override
Project.csv,  ProjectType, 1, 8
```

Reply:
Huh, yeah, I guess I got overly excited about making it read well.  I'll revisit.  Thanks.

Reply:
Updated, with updated screenshots above.